android/Omnivore/app/src/main/java/app/omnivore/omnivore/feature/onboarding/auth/AuthProviderScreen.kt

```diff
@@ -27,28 +26,25 @@ import app.omnivore.omnivore.feature.onboarding.OnboardingViewModel
 import app.omnivore.omnivore.feature.onboarding.auth.provider.AppleAuthButton
 import app.omnivore.omnivore.feature.onboarding.auth.provider.GoogleAuthButton
 import app.omnivore.omnivore.navigation.Routes
-import com.google.android.gms.common.GoogleApiAvailability
 
 @Composable
 fun AuthProviderScreen(
     welcomeNavController: NavHostController,
     viewModel: OnboardingViewModel
 ) {
-    val isGoogleAuthAvailable: Boolean =
```

Comment:
Do we have to remove this? We have some cases where the user installs not through Play.

Reply:
It was checking if google play services is available or not. It wasn't checking if the app is installed from Google Play.
Someone can still install it from somewhere else and still has Google play services.